include/aws/crt/mqtt/Mqtt5Listener.h

```diff
@@ -0,0 +1,344 @@
+#pragma once
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+#include <aws/crt/mqtt/Mqtt5Client.h>
+#include <aws/crt/mqtt/Mqtt5Types.h>
+#include <aws/mqtt/v5/mqtt5_listener.h>
+
+namespace Aws
+{
+    namespace Crt
+    {
+        namespace Mqtt5
+        {
+            class Mqtt5ListenerOptions;
+
+            /**
+             * Type signature of the callback invoked when a PacketPublish message received for Mqtt5ListenerCore.
+             *
+             * @return true if the message get handled otherwise return false
+             */
+            using OnListenerPublishReceivedHandler = std::function<bool(const PublishReceivedEventData &)>;
+
+            /**
+             * An MQTT5 listener core. The core is only internally used by Mqtt5Listener to manage the callbacks and
```

Comment:
trivial
"The core is only used internally by Mqtt5Listener to manage the callbacks and underlying c objects."